Dockerfile-dc

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.13-alpine
+
+RUN apk add -U --no-cache ca-certificates
+
+ADD . /app
+WORKDIR /app
+ENV GO111MODULE on
+RUN go get -u github.com/gobuffalo/packr/v2/packr2
+RUN packr2
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build
+
+FROM scratch
+
+COPY --from=0 /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=0 /app/oathkeeper /usr/bin/oathkeeper
+COPY --from=0 /app/.docker_compose/config.yaml config.yaml
```

Comment:
As I said in the summary comment, adding config files as part of the build process is bad practice. Let's remote these. I've created another image for another project which makes use of volumes to mount the config. It uses a standard alpine-based image with a custom user: https://github.com/ory/kratos/blob/master/.docker/Dockerfile

Then, we simply [mount the folder where the config files are at](https://github.com/ory/kratos/blob/master/quickstart.yml#L82-L85) and use the [config flag](https://github.com/ory/kratos/blob/master/quickstart.yml#L75) to define where the config files is saved.

I think a 1:1 copy should work more or less for oathkeeper too. The cool thing is, hot reloading will work! So you can edit your config file on the host and oathkeeper will reload it!

Reply:
You're right, I'm just wondering if there's a better way to even add the private keys. I've made the changes you've suggested here.